internal/helm/common/string_resource.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2022 The Flux authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import "strings"
+
+// stringResource is there to satisfy the github.com/google/go-containerregistry/pkg/authn.Resource interface.
```

Comment:
```suggestion
// StringResource is there to satisfy the github.com/google/go-containerregistry/pkg/authn.Resource interface.
```

Also it would be nice to comments the methods

Reply:
Resolved in [9347bdc](https://github.com/fluxcd/source-controller/pull/1075/commits/9347bdcd0d0e9a89120c12ae804f8eb4bfd46f3a).